portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -13,20 +13,64 @@
 // limitations under the License.
 
 package com.google.sps.servlets;
+import com.google.sps.data.Comment;
+import com.google.gson.Gson;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.SortDirection;
 
 import java.io.IOException;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.*;  
```

Comment:
The Google Java style guide doesn't like wildcard imports.

Reference: https://google.github.io/styleguide/javaguide.html#s3.3.1-wildcard-imports

Reply:
Removed the * import and imported necessary packages